geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/ClientServerConnectDisconnectDistributedTest.java

```diff
@@ -151,9 +151,9 @@ private void verifyServerConnectionSubjectsAreLoggedIn(AcceptorImpl acceptor) {
     authorizations = new ArrayList<>();
     for (ServerConnection sc : acceptor.getAllServerConnections()) {
       ClientUserAuths auth = sc.getClientUserAuths();
-      assertThat(auth.getSubjects().size()).isNotEqualTo(0);
+      assertThat(auth.getAllSubjects().size()).isNotEqualTo(0);
```

Comment:
The failure message will show the contents of the collection if you change to:
```
assertThat(auth.getAllSubjects()).isEmpty();
```

Reply:
done